plutus-pab/tx-inject/config.yaml

```diff
@@ -3,16 +3,16 @@ dbConfig:
     dbConfigPoolSize: 20
 
 pabWebserverConfig:
-  baseUrl: http://localhost:8080
+  baseUrl: http://localhost:9080
```

Comment:
Why change the ports?

Reply:
I was wondering wether to keep this change. I think `8080` is used quite a bit by user permission web servers, especially when developing some web applications. I can remove it if you don't like it.

Reply:
I don't mind either way, but if you keep it please update `plutus-pab-client/pab-demo-scripts.nix` as well.

Reply:
and `plutus-pab/plutus-pab.yaml.sample`